rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -641,20 +641,52 @@ namespace rsx
 #endif
 			u64 start_time = get_system_time();
 
+			// Vblank rate, these are updated on the first iteration
+			u64 vblank_rate = g_cfg.video.vblank_rate;
+			u64 vblank_period = 1'000'000 + u64{g_cfg.video.vblank_ntsc.get()} * 1000;
+
+			u64 local_vblank_count = 0;
+
+			auto multiply_by_ratio = [](u64 source, u64 numerator, u64 denominator) -> u64
```

Comment:
i don't really see a reason for this to be a function anymore

Reply:
Removed.